devices/oneplus-oneplus2/kernel/default.nix

```diff
@@ -0,0 +1,6 @@
+{
+    import = [
+        # FIXME(Krey): Integrate logic to use kernel on demand
+        ./stealth/default.nix
+    ]
+}
```

Comment:
With provenance, this will not be needed.

Reply:
provenance?